client/src/components/pages/ProfileUpdate.tsx

```diff
@@ -86,9 +86,38 @@ const Screen: FC = () => {
   const [profilePath, setProfilePath] = useState('');
   const environment = useRelayEnvironment();
   const envrionmentProps = useRef(environment);
+  const snackbar = useRef<SnackbarRef>(null);
 
   const {user} = useAuthContext();
 
+  const UpdateProfile = useCallback(
```

Comment:
The name of `function` should be written in `lowerCamelCase`.

See [import part in dooboo-ui contributing guide](https://dooboo-ui.dooboolab.com/?path=/story/contributing-contributing--page).

<img width="809" alt="Screen Shot 2021-08-24 at 11 47 57 PM" src="https://user-images.githubusercontent.com/27461460/130638387-e4135a4e-5101-4d1f-af74-359167209df4.png">